### PR TITLE
srcset support #8623

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: modules/portal/portal-api/src/main/java/com/enonic/xp/portal/url/ProcessHtmlParams.java [hunk 1]
```diff
@@ -23,11 +29,26 @@ public ProcessHtmlParams value( final String value )
...
+
     @Override
     public ProcessHtmlParams setAsMap( final Multimap<String, String> map )
     {
         super.setAsMap( map );
         value( singleValue( map, "_value" ) );
+        imageWidths( Objects.requireNonNullElse( map.removeAll( "_imageWidths" ), List.<String>of() ).
+            stream().
+            map( Integer::parseInt ).
+            collect( Collectors.toList() ) );
```
Comment (rymsha) [inline modules/portal/portal-api/src/main/java/com/enonic/xp/portal/url/ProcessHtmlParams.java:51]: Use unmodifiable

File: modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/ProcessHtmlHandler.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
 public final class ProcessHtmlHandler
     extends AbstractUrlHandler
 {
-    private static final Set<String> VALID_URL_PROPERTY_KEYS = new HashSet<>( Arrays.asList( "value", "type" ) );
+    private static final Set<String> VALID_URL_PROPERTY_KEYS = new HashSet<>( Arrays.asList( "value", "type", "imageWidths" ) );
```
Comment (rymsha) [inline modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/ProcessHtmlHandler.java:14]: Set.of

File: modules/portal/portal-api/src/main/java/com/enonic/xp/portal/url/ProcessHtmlParams.java (hunk 1 above)
Comment (anatol-sialitski) [reply modules/portal/portal-api/src/main/java/com/enonic/xp/portal/url/ProcessHtmlParams.java]: done

File: modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/ProcessHtmlHandler.java (hunk 2 above)
Comment (anatol-sialitski) [reply modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/ProcessHtmlHandler.java]: done

**Review by rymsha — APPROVED** [review]

**Pull request merged**